packages/rancher-alerting-drivers/charts/Chart.yaml

```diff
@@ -0,0 +1,44 @@
+name: rancher-alerting-drivers
+home: ""
+sources: []
+version: 1.0.1
+description: The manager for third-party webhook receivers used in the Alertmanger
+keywords:
+- monitoring
+- alertmanger
+- webhook
+maintainers:
+- name: Jiaqi Luo
+  email: ""
+  url: https://github.com/jiaqiluo
+icon: ""
+apiversion: v2
+condition: ""
+tags: ""
```

Comment:
A lot of this stuff can be removed, let's keep this as simple as possible. Unless its required or relevant I wouldn't add.

Reply:
This file is updated by the chart-build-script, I will try to keep it compact before submitting the PR.

Reply:
Does scripts add stuff back in ? if not I would remove icon, tags, condition, home etc...

Reply:
yes, all those are added back after running the `make prepare`.

As I know the new version of the chart-build-script will not add those empty fields.

Reply:
Yes, the current scripts adds back in null entries while the new YAML marshaller does not do this